packages/statemanager/test/proofStateManager.spec.ts

```diff
@@ -28,6 +29,20 @@ describe('ProofStateManager', () => {
     assert.equal(proof.nonce, '0x0', 'Nonce is in quantity-encoded RPC representation')
   })
 
+  it(`should correctly return the right storage root / account root`, async () => {
```

Comment:
This tests that the trie is flushed in`getProof`. If the trie is not flushed, then the proof will report storage values but it will not report the right storage hash (storage root).